resources/hab_build.rb

```diff
@@ -67,6 +68,12 @@ def last_build_env
     retries new_resource.retries
     live_stream new_resource.live_stream
   end
+
+  execute 'remove-studio' do
+    command "sudo -E #{hab_binary} studio rm #{hab_studio_path}"
+    live_stream new_resource.live_stream
+    only_if { new_resource.cleanup }
```

Comment:
Generally it is preferred style in custom resources to use the property, not `new_resource.property`. Thus, use `cleanup`, not `new_resource.cleanup`.

Ref https://docs.chef.io/custom_resources.html#new-resource-property